openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -186,6 +190,15 @@ public boolean isHttpOROAuth() {
         return httpOROAuth;
     }
 
+    /**
+     * Returns `true` if tokenUrl value available in OAuth clientCredentials.
+     *
+     * @return {@link boolean}
+     */
+    public boolean isClientCredGrantTokenUrlAvailable() {
```

Comment:
Now we can remove this function since it does not have any outside usages. WDYT ?

Reply:
+1. Removed